Jenkinsfile.internal

```diff
@@ -41,6 +41,9 @@ pipeline {
                     }
                 }
                 stage('Container Tests') {
+                    agent {
```

Comment:
so the other one is running somewhere it was defined. But the main pipeline is running on our controller and waits for the systemstests anyway. So it blocks the controller. Maybe it is better not to wait for completion and set the flag to false but still have to handle the results.